corehq/apps/userreports/data_source_providers.py

```diff
@@ -13,7 +13,10 @@ def get_data_sources(self):
 class DynamicDataSourceProvider(DataSourceProvider):
 
     def get_data_sources(self):
-        return [config for config in DataSourceConfiguration.all() if not config.is_deactivated]
+        return [DataSourceConfiguration.wrap(r['doc'])
```

Comment:
What datatype does `DataSourceConfiguraton.get_db().view(...)` return? I thought that it returns a wrapped object, which would mean that this line wraps the object twice. Could it be changed to `return DataSourceConfiguration.get_db().view(...).all()` ?

Reply:
It would be great if there was a way to avoid that wrap. For now when i fetch
`active_confs = DataSourceConfiguration.get_db().view('userreports/active_data_sources', include_docs=True,reduce=False).all()`
the docs i get a JSON.

Reply:
if you just did `DataSourceConfiguration.view` instead of `DataSourceConfiguration.get_db().view` you'd get wrapped docs.